rollup.config.js

```diff
@@ -19,8 +23,41 @@ const processEnv = conf => ({
   'process.env': `(${JSON.stringify(conf)})`,
 });
 
-const vuePlugin = process.env.VUE_VERSION === 'vue3' ? vueV3 : vueV2;
-const outputDir = process.env.VUE_VERSION === 'vue3' ? 'vue3' : 'vue2';
+const vuePlugin = isVue3 ? vueV3 : vueV2;
+const outputDir = isVue3 ? 'vue3' : 'vue2';
+
+const excludeCompositionsAPI = {
+  load(id) {
+    if (id.endsWith('/src/compositions/index.js')) {
+      return '';
+    }
+    return null;
+  },
+};
+
+const applyVueCompat = {
+  load(id) {
+    if (id.endsWith('src/util/vue-compat/index.js')) {
+      if (isVue2) {
+        return fs.readFileSync('./src/util/vue-compat/index-2.js').toString();
+      } else if (isVue3) {
+        return fs.readFileSync('./src/util/vue-compat/index-3.js').toString();
+      }
+    }
+    return null;
+  },
+};
+
+const applyVueCompatForVue2Compositions = {
```

Comment:
When bundling `vue-instantsearch/compositions` for vue 2, we need a different file for `vue-compat` util because we need to bind `@vue/compositions` api.